firmware/quadruna/VC/src/tasks.c

```diff
@@ -40,12 +49,98 @@ void canTxQueueOverflowCallBack(uint32_t overflow_count)
     .rx_overflow_callback = canRxQueueOverflowCallBack,
 };
 
+static const Gpio      buzzer_pwr_en    = { .port = BUZZER_PWR_EN_GPIO_Port, .pin = BUZZER_PWR_EN_Pin };
+static const Gpio      bat_i_sns_nflt   = { .port = BAT_I_SNS_NFLT_GPIO_Port, .pin = BAT_I_SNS_NFLT_Pin };
+static const BinaryLed led              = { .gpio = { .port = LED_GPIO_Port, .pin = LED_Pin } };
+static const Gpio      telem_pwr_en     = { .port = TELEM_PWR_EN_GPIO_Port, .pin = TELEM_PWR_EN_Pin };
+static const Gpio      npcm_en          = { .port = NPCM_EN_GPIO_Port, .pin = NPCM_EN_Pin };
+static const Gpio      acc_i_sense_nflt = { .port = ACC_I_SENSE_NFLT_GPIO_Port, .pin = ACC_I_SENSE_NFLT_Pin };
+static const Gpio      pgood            = { .port = PGOOD_GPIO_Port, .pin = PGOOD_Pin };
+static const Gpio      lv_pwr_en        = { .port = LV_PWR_EN_GPIO_Port, .pin = LV_PWR_EN_Pin };
+static const Gpio      aux_pwr_en       = { .port = AUX_PWR_EN_GPIO_Port, .pin = AUX_PWR_EN_Pin };
+static const Gpio      pump_pwr_en      = { .port = PUMP_PWR_EN_GPIO_Port, .pin = PUMP_PWR_EN_Pin };
+static const Gpio      nchrg_fault      = { .port = NCHRG_FAULT_GPIO_Port, .pin = NCHRG_FAULT_Pin };
+static const Gpio      nchrg            = { .port = NCHRG_GPIO_Port, .pin = NCHRG_Pin };
+static const Gpio      inv_l_pwr_en     = { .port = INV_L_PWR_EN_GPIO_Port, .pin = INV_L_PWR_EN_Pin };
+static const Gpio      inv_r_pwr_en     = { .port = INV_R_PWR_EN_GPIO_Port, .pin = INV_R_PWR_EN_Pin };
+static const Gpio      fr_stby1         = { .port = FR_STBY1_GPIO_Port, .pin = FR_STBY1_Pin };
+static const Gpio      fr_stby2         = { .port = FR_STBY2_GPIO_Port, .pin = FR_STBY2_Pin };
+static const Gpio      fr_stby3         = { .port = FR_STBY3_GPIO_Port, .pin = FR_STBY3_Pin };
+static const Gpio      inv_l_program    = { .port = INV_L_PROGRAM_GPIO_Port, .pin = INV_L_PROGRAM_Pin };
+static const Gpio      inv_r_program    = { .port = INV_R_PROGRAM_GPIO_Port, .pin = INV_R_PROGRAM_Pin };
+static const Gpio      l_shdn_sns       = { .port = L_SHDN_SNS_GPIO_Port, .pin = L_SHDN_SNS_Pin };
+static const Gpio      r_shdn_sns       = { .port = R_SHDN_SNS_GPIO_Port, .pin = R_SHDN_SNS_Pin };
+static const Gpio      nchimera         = { .port = NCHIMERA_GPIO_Port, .pin = NCHIMERA_Pin };
+static const Gpio      nprogram_3v3     = { .port = NPROGRAM_3V3_GPIO_Port, .pin = NPROGRAM_3V3_Pin };
+static const Gpio      sb_ilck_shdn_sns = { .port = SB_ILCK_SHDN_SNS_GPIO_Port, .pin = SB_ILCK_SHDN_SNS_Pin };
+static const Gpio      tsms_shdn_sns    = { .port = TSMS_SHDN_SNS_GPIO_Port, .pin = TSMS_SHDN_SNS_Pin };
+
+static const Gpio *id_to_gpio[] = {
+    [GpioNetName_BUZZER_PWR_EN]    = &buzzer_pwr_en,
+    [GpioNetName_BAT_I_SNS_NFLT]   = &bat_i_sns_nflt,
+    [GpioNetName_LED]              = &led.gpio,
+    [GpioNetName_TELEM_PWR_EN]     = &telem_pwr_en,
+    [GpioNetName_NPCM_EN]          = &npcm_en,
+    [GpioNetName_ACC_I_SENSE_NFLT] = &acc_i_sense_nflt,
```

Comment:
pls run "fix_formatting.py" in "Consolidated-Firmware/scripts/clang_format" before merging when changing C files